tests/SprykerEcoTest/Zed/Unzer/_support/UnzerZedTester.php

```diff
@@ -0,0 +1,565 @@
+use Generated\Shared\Transfer\UnzerPaymentTransfer;
+use Spryker\Shared\Vault\VaultConstants;
+use SprykerEco\Shared\Unzer\UnzerConstants;
+use SprykerEco\Zed\Unzer\Business\UnzerFacadeInterface;
+use SprykerEco\Zed\Unzer\Persistence\UnzerEntityManager;
+use SprykerEco\Zed\Unzer\Persistence\UnzerRepository;
+use SprykerEco\Zed\Unzer\UnzerConfig;
+
+/**
+ * Inherited Methods
+ *
+ * @method void wantToTest($text)
+ * @method void wantTo($text)
+ * @method void execute($callable)
+ * @method void expectTo($prediction)
+ * @method void expect($prediction)
+ * @method void amGoingTo($argumentation)
+ * @method void am($role)
+ * @method void lookForwardTo($achieveValue)
+ * @method void comment($description)
+ * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = NULL)
+ *
+ * @SuppressWarnings(PHPMD)
+ */
+class UnzerZedTester extends Actor
+{
+    use _generated\UnzerZedTesterActions;
+
+    /**
+     * @var string
+     */
+    public const PAYMENT_SELECTION_MARKETPLACE = 'unzerMarketplaceBankTransfer';
+
+    /**
+     * @var int
+     */
+    public const TOTALS_PRICE_TO_PAY = 72350;
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_COUNTRY = 'Germany';
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_CITY = 'Berlin';
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_ZIP = '20537';
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_ADDITIONAL = 'street';
+
+    /**
+     * @var int
+     */
+    public const ADDRESS_NO = 324324;
+
+    /**
+     * @var string
+     */
+    public const ADDRESS_STREET = 'Street';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_SALUTATION = 'Mr';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_SURNAME = 'Doe';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_NAME = 'John';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_EMAIL = 'john.doe@mail.com';
+
+    /**
+     * @var string
+     */
+    public const CUSTOMER_REFERENCE = 'DE-22';
+
+    /**
+     * @var string
+     */
+    public const PAYMENT_METHOD = 'unzerMarketplaceBankTransfer';
+
+    /**
+     * @var string
+     */
+    public const PAYMENT_PROVIDER = 'unzer';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_CUSTOMER_ID = 's-cust-43434234';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_METADATA_ID = 's-meta-5423423';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_BASKET_ID = 's-bskt-435345';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_PAYMENT_RESOURCE_ID = 's-type-sofort-1';
+
+    /**
+     * @var string
+     */
+    public const UNZER_API_RESPONSE_WEBHOOK_URL = 'https://unzer-spryker.com';
+
+    /**
+     * @var string
+     */
+    public const UNZER_KEYPAIR_ID = 'key-1';
+
+    public const UNZER_MAIN_REGULAR_KEYPAIR_ID = 'keypair-id-1';
```

Comment:
```suggestion
    /**
     * @var string
     */
    protected const UNZER_MAIN_REGULAR_KEYPAIR_ID = 'keypair-id-1';
```

Please check similar cases.